update-header/action.yaml

```diff
@@ -0,0 +1,70 @@
+name: "Update File headers to the new year"
+author: "Jaspar Löchte <jaspar.loechte@greenbone.net>"
+description: "An action that updates header year in all files of given directories."
+
+inputs:
+  github-user:
+    description: "Github user name."
+    required: true
+  github-user-mail:
+    description: "Mail address for the given github user."
+    required: true
+  github-user-token:
+    description: "Token with write rights required to create the release."
+    required: true
+  version:
+    description: "Python version that should be installed"
+    default: 3.9
+  target:
+    description: "Branch that should be released. Default: main"
+    default: 'main'
+  directories:
+    description: "Specify the directories that should be updated"
```

Comment:
add `required: false`
add `default: '.'`

Reply:
I am not sure about `.`, because many files e.g. `.txt` files will get a header, that maybe shall not get one. Therefore we would need to have an absolute clean `.pontos-header-ignore` file in every repository.

Reply:
but because of not default its empty and this would make problems in the composite step later.

I would add a step that checks if a `.pontos-header-ignore` is exist else fail.

Reply:
pontos-update-header is already caring about that file if empty or not existing it just doesn't exclude any files